rdkit/Chem/Fingerprints/ClusterMols.py

```diff
@@ -70,7 +72,7 @@ def ClusterPoints(data, metric, algorithmId, haveLabels=False, haveActs=True,
     acts = [int(x[2]) for x in data]
 
   if not haveLabels:
-    labels = ['Mol: %s' % str(x[0]) for x in data]
+    labels = [f'Mol: {str(x[0])}' for x in data]
```

Comment:
You don't need to conversion to `str`, this will happen automatically.

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.